src/transformers/pytorch_utils.py

```diff
@@ -277,3 +277,12 @@ def meshgrid(
         if indexing != "ij":
             raise ValueError('torch.meshgrid only supports `indexing="ij"` for torch<1.10.')
         return torch.meshgrid(*tensors)
+
+
+def id_tensor_storage(tensor: torch.Tensor) -> Tuple[torch.device, int, int]:
```

Comment:
But this is expected no? This is an identifier for a tensor storage and not of a tensor. Some different tensors can still share the same storage?

Reply:
Ok. We had multiple bugs in the load due to a similar function considering all tensors on the meta device were tied, but if this is only used for the save it can work I guess. Can you make it more explicit in the docs it should only be used for that?

Reply:
Add a comment about the identifier being storage specific + example with "meta"